samples/modules/lvgl/README.rst

```diff
@@ -0,0 +1,47 @@
+.. _lvgl_demo_sample:
+
+LVGL demos sample
+###################
+
+Overview
+********
+
+A sample showcasing upstream LVGL demos.
+
+* Music: The music player demo shows what kind of modern, smartphone-like user interfaces can be created on LVGL.
+* Benchmark: The benchmark demo tests the performance in various cases. For example rectangle, border, shadow, text, image blending, image transformation, blending modes, etc.
+* Stress: A stress test for LVGL. It contains a lot of object creation, deletion, animations, styles usage, and so on. It can be used if there is any memory corruption during heavy usage or any memory leaks.
+
+Requirements
+************
+
+* A board with display, idealy with 480x272 resolution
```

Comment:
```suggestion
* A board with display, ideally with 480x272 resolution
```

is a higher resolution ok too? if yes should this read 

```suggestion
* A board with display, ideally with 480x272 resolution or higher.
```

Reply:
Should be, even with the recommended resolution there are some label eliding issues.